### ISSUE TITLE
keystrokes missed when autocomplete is guessing

### ISSUE THREAD

**Issue by doctorcolossus** [scrapmac/User-Style-Manager#134]
When I start typing quickly and autocomplete is trying to guess what I might want to type, I've noticed that keystrokes are often omitted, so that I have to backspace and begin again more slowly. I'll give you a real example. Just now, I fired up User Style Manager and typed "{display:none;}", but the result was "{dplay: n;}". Then I tried with just "display: none" and got "dsplay noe". This is terrible. Autocomplete is nice, but really not necessary for me, so it is kind of impeding my workflow. Another thing is that it has been automatically inserting colons and closing brackets for me when I didn't expect it, causing me to commit syntax errors I wouldn't have otherwise made.

I'm using the i3 window manager. I wonder if it might be that an entire new window is created for autocomplete, which momentarily steals the input focus.

**Comment by Gitoffthelawn** [comment]
It would be nice to have an option to just turn off autocomplete at times, or have it on-demand, where you need a keystroke to make suggestions for specific text.

**Comment by doctorcolossus** [comment]
I totally agree. It seems like Girish has gone MIA though. I hope he's OK. Maybe he's just busy. Anyway, I guess one of us should get to work on a patch.
